### PR TITLE
Disambiguate type exports

### PR DESCRIPTION
**Pull request by KnorpelSenf**
I don't want to make people think that they need to install a new package. Some folks refuse to read on so they miss the code examples.

### CODE REVIEW

File: site/docs/guide/api.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ grammY allows you to specify objects consistently across the API, and makes sure
 
 grammY ships with complete type coverage of the Bot API.
 The [`@grammyjs/types`](https://github.com/grammyjs/types) repository contains the type definitions that grammY uses internally.
-These type definitions are also exported so you can use them in your own code.
+These type definitions are also directly exported from the `grammy` core package so you can use them in your own code.
```
Comment (roj1512) [inline site/docs/guide/api.md:80]:
```suggestion
These type definitions are also directly exported from the core `grammy` package so you can use them in your own code.
```

Comment (KnorpelSenf) [reply site/docs/guide/api.md:80]: I think both versions work but they express different things, no?

Comment (roj1512) [reply site/docs/guide/api.md:80]: My interpretation:

- _`grammy` core package_ is like _`grammy` is not the only core package_.
- _core `grammy` package_ is like _the core package aka `grammy`_.

What’s yours?

Comment (KnorpelSenf) [reply site/docs/guide/api.md:80]: I think we have the same definition. My intent was to say _core package_ and then specify the name of the package, so it's the _`grammy` core package_. But I think people will know that we mean either way so we can call it the _core `grammy` package_, too, even though it sounds to me like there were several packages called `grammy` and we specify that we mean the core one.

Comment (KnorpelSenf) [reply site/docs/guide/api.md:80]: I'll leave it up to you, feel free to accept your own suggestion if you want to go ahead

**Review by roj1512 — APPROVED** [review]

**Review by LWJerri — APPROVED** [review]

**Review by fadzikri — APPROVED** [review]

**Review by habemuscode — APPROVED** [review]

**Pull request merged**